### DISCUSSION TITLE
TLS Issue

### DISCUSSION THREAD

**Discussion by WPTechnician** [snipe/snipe-it#12549] [Q&A]
Hello, I am new to SnipeIT and I am excited to get it into use.  I am having an issue with the URL.  I need it to use https://server.domain.com.  I have updated my .env to reflect that but when I hit the URL I am not getting any stylesheets and looking at the console I see:

![image](https://user-images.githubusercontent.com/19175877/220709305-01dddd59-2a40-444d-9cc6-e168e3b49141.png)

Is there something else I am missing?  Thanks!

**Comment by snipe** [comment]
Maybe try clearing your env cache? `php artisan config:clear`? Are you running through a proxy or anything?

**Reply by WPTechnician** [reply]
Hi, yes!  I am running through a proxy.  And I just found the reverse proxy document that explains it all exactly.  Thanks for responding, I will go and hide now.  :-)
